mteb/models/e5_instruct.py

```diff
@@ -113,6 +113,7 @@
     similarity_fn_name="cosine",
     use_instructions=True,
     training_datasets={
+        **E5_MISTRAL_TRAINING_DATA,
```

Comment:
Maybe in the future the `adapted_from` could automatically infer this

Reply:
Yep, agree. This should just happen automatically